src/sidebar/services/annotations-exporter.ts

```diff
@@ -0,0 +1,39 @@
+import type { Annotation } from '../../types/api';
+import { VersionData } from '../helpers/version-data';
+import type { SidebarStore } from '../store';
+
+export type ExportContent = {
+  export_date: string;
+  export_userid: string;
+  client_version: string;
+  annotations: Annotation[];
+};
+
+/**
+ * Generates annotations exports
+ *
+ * @inject
+ */
+export class AnnotationsExporter {
+  private _store: SidebarStore;
+
+  constructor(store: SidebarStore) {
+    this._store = store;
+  }
+
+  /**
+   * @param now - Test seam
+   */
+  buildExportContent(now = new Date()): ExportContent {
+    const profile = this._store.profile();
+    const annotations = this._store.allAnnotations();
+    const versionData = new VersionData(profile, []);
```

Comment:
I haven't found a better way to properly get the client version, other than creating a new `VersionData` instance.

The main problem is that this class wraps more information and logic, but at the same time, it is the single place where the `__VERSION__` placeholder is defined, ensuring it will be replaced at build time with the actual version.

Another option would be to make this service define its own property with `__VERSION__` value.

Reply:
`__VERSION__` is a placeholder that gets replaced by Rollup. It can appear in any file. See references to `__VERSION__` in `rollup.config.mjs`.

Reply:
Yep, that's why I mentioned this.

> Another option would be to make this service define its own property with __VERSION__ value.

What I don't know is if it would be desirable or not.

Reply:
Yep, that's exactly my thinking.

In fact, I passed `profile` there just because I had it for free, but the truth is that anything would still work as we are just reading `versionData.version` here.